File: UCF HSPT Project/src/y2008/ucfbucks.java
// Necessary Imports
import java.util.*;
import java.io.*;

public class ucfbucks {
	
	public static void main (String[] args) throws IOException {
		
		// Make a Scanner object that will read in from ucfbucks.in
		Scanner in = new Scanner (new File("ucfbucks.in"));
		
		// Read in the Number of Purchases.
		int numberOfPurchases = in.nextInt();
		
		// Loop as many times as there are purchases.
		for (int index = 1; index <= numberOfPurchases; index++){
			
			// Read in the amount of ucf dollars you have
			int cashOnHand = in.nextInt();
			
			// Read in the cost of the item
			double cost = in.nextDouble();
			
			// Print out the difference of the ceiling of the cost and the cost casted as an integer
			System.out.println((int)((Math.ceil(cost)-cost)*100));
			
		}//for
		
	} // main
	
} // ucfbucks

File: UCF HSPT Documents/2008/Solutions/ucfbucks.cpp
// Necessary namespace and includes
#include <iostream>
#include <fstream>
#include <string>
#include <math.h>

using namespace std;

int main(){
	
	// variable to hold the number of names in the file
	int numberOfPurchases;
	
	// open the file ucfbucks.in
	ifstream fin("ucfbucks.in");
	 
	// Read in the Number of Names.
	fin >> numberOfPurchases;
	
	// Loop as many times as there are names.
	for (int index = 1; index <= numberOfPurchases; index++) {
		
		// declare some variables to use later
		int cashOnHand;
		double cost;
		
		// Read in the cashOnHand and the cost
		fin >> cashOnHand >> cost;
		
        // Print out the difference of the ceiling of the cost and the cost casted as an integer
		cout << (int)((ceil(cost)-cost)*100) << endl;
		
	}// for
	
	// finish up
	return 0;
	
}// Main
